libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/RustMatrixClient.kt

```diff
@@ -536,9 +467,9 @@ class RustMatrixClient(
         deleteSessionDirectory(deleteCryptoDb = false)
     }
 
-    override suspend fun logout(ignoreSdkError: Boolean): String? = doLogout(
-        doRequest = true,
-        removeSession = true,
+    override suspend fun logout(ignoreSdkError: Boolean, forced: Boolean): String? = doLogout(
```

Comment:
now that `doLogout` is called only once, I guess there is no need to have a sub private method anymore. Also the `forced` parameter is maybe a bit hard to understand, maybe rename to `from401` or something better?

Reply:
I have inlined the doLogout call and `forced` is now inverted and called `userInitiated`.

Reply:
Nicer, thanks!